app/Http/Middleware/RequestLogger.php

```diff
@@ -16,17 +23,15 @@ class RequestLogger
      */
     public function handle($request, Closure $next)
     {
-        $filepath = base_path(self::LOG_FILEPATH);
-
         $log_message = "Request: ".sprintf(
             '%s %s %s',
             $request->getMethod(),
             $request->getRequestUri(),
             json_encode($request->all())
         )."\n";
 
-        file_put_contents($filepath, $log_message, FILE_APPEND);
-
+        $this->logger->info($log_message);
```

Comment:
Here we just log the message, we don't need to worry about the filepath or filesystem details, we're relying on the logger to take care of that.